pkg/types/openstack/machinepool.go

```diff
@@ -22,9 +22,14 @@ type MachinePool struct {
 	// where each ID is presented in UUID v4 format.
 	// +optional
 	AdditionalSecurityGroupIDs []string `json:"additionalSecurityGroupIDs,omitempty"`
+
+	// Zones contains a list of availabilty zones that instances can be deployed on
```

Comment:
This sentence is weird.
```suggestion
	// Zones is the list of availability zones where the instances should be deployed.
	// Instances in this pool will be deployed to all the zones possible, spreading the required replicas evenly
	// across all of them.
```